src/dat.c

```diff
@@ -228,7 +229,7 @@ int eraDat(int iy, int im, int id, double fd, double *deltat )
    if (i < 0) return -5;
 
 /* Get the Delta(AT). */
-   da = changes[(i < 0) ? 0 : i].delat;
+   da = changes[i].delat;
```

Comment:
Exactly. That's why I only kept the first check for a negative number. 

The change was added in 506f3027f3cdf382480af7fbc5e4bb4589ba56de which is part of the previous SOFA release. It looks to me like they edited the current `dat.c` from an old version of the source and not the "2013-12-02_d" release. The version of `dat.c` here does include the previous patch and the new version and correctly drops the local ERFA hack. I'll send a note to see if the dropping of the `-5` error condition was deliberate. I will also update the README to say that the local patch is no longer needed (at least in the stated form).